dags/openshift_nightlies/manifest.yaml

```diff
@@ -57,10 +65,21 @@ platforms:
       - name: aws
         profiles:
         - default
-        - ovn
+        - sdn-common-control-plane
```

Comment:
I thought we were going to do sdn-common-[cp|dp].

Reply:
done